src/main/java/uk/gov/hmcts/dts/fact/services/CourtService.java

```diff
@@ -80,6 +80,14 @@ public Court getCourtBySlug(final String slug) {
             .orElseThrow(() -> new NotFoundException(slug));
     }
 
+    public List<Court> getCourtsByCourtTypes(final List<String> courtTypes) {
```

Comment:
Think we said we would do docstrings for each method in retro? But yeah this ones self explanatory lol